packages/flutter/lib/src/material/date_picker_theme.dart

```diff
@@ -165,14 +165,64 @@ class DatePickerThemeData with Diagnosticable {
   /// grid of the date picker.
   ///
   /// This will be used instead of the color provided in [dayStyle].
+  ///
+  /// This supports different colors based on the [WidgetState]s of
+  /// the day button, such as `WidgetState.selected`, `WidgetState.hovered`,
+  /// `WidgetState.focused`, and `WidgetState.disabled`.
```

Comment:
Should we expect to add a comment like this on every WidgetStateProperty?

Reply:
I think it would be good if we could clarify which WidgetStates are supported in doc:)

Reply:
If so, then should this comment be more precise instead of saying "such as"? That makes it seem like it might support other WidgetStates too.